data/tribes/initialization/empire/units.lua

```diff
@@ -1522,6 +1778,12 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_tower",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire military site: Tower
+            lore = pgettext("empire_building", "‘Veni, Vidi, Vici!’<br>(I came, I saw, I conquered!)"),
+            -- TRANSLATORS: Lore author helptext for an empire military site: Tower
+            lore_author = pgettext("empire_building",
+               "General Claudius Cæcus Cessus reporting a quick victory over the Barbarians thanks to good "..
+               "reconnaissance provided by the towers on the border"),
```

Comment:
could be shortened: 
"General Claudius Cæcus Cessus reporting a quick victory over the Barbarians thanks to superior "..
               "vision of a tower"),